packages/adapters/solana/src/client.ts

```diff
@@ -433,50 +372,10 @@ export class SolanaAdapter extends AdapterBlueprint {
   public async syncConnection(
     params: AdapterBlueprint.SyncConnectionParams
   ): Promise<AdapterBlueprint.ConnectResult> {
-    const { id, rpcUrl } = params
-    const connector = this.connectors.find(c => c.id === id)
-    const selectedProvider = connector?.provider as Provider
-
-    if (!selectedProvider) {
-      throw new Error('Provider not found')
-    }
-
-    // Handle different provider types
-    if (connector?.type === 'AUTH') {
-      const authProvider = selectedProvider as unknown as W3mFrameProvider
-      const user = await authProvider.getUser({
-        chainId: Number(this.caipNetworks?.[0]?.id)
-      })
-
-      if (!user?.address) {
-        throw new Error('No address found')
-      }
-
-      return {
-        address: user.address,
-        chainId: typeof user.chainId === 'string' ? Number(user.chainId.split(':')[1]) : 1,
-        provider: selectedProvider,
-        type: connector.type,
-        id
-      }
-    }
-
-    // For standard Solana wallets
-    const address = await selectedProvider.connect()
-    const { chainId: activeChainId } = StorageUtil.getActiveNetworkProps()
-    const chainId = activeChainId || solana.id
-
-    this.listenProviderEvents(selectedProvider as unknown as WalletStandardProvider)
-
-    SolStoreUtil.setConnection(new Connection(rpcUrl, 'confirmed'))
-
-    return {
-      address,
-      chainId,
-      provider: selectedProvider,
-      type: connector?.type as ConnectorType,
-      id
-    }
+    return this.connect({
+      ...params,
+      type: ''
```

Comment:
no type needed ?

Reply:
no type needed here as it's reusing the connect method for its data.